lib/ae_mdw/db/sync/name.ex

```diff
@@ -24,7 +30,19 @@ defmodule AeMdw.Db.Sync.Name do
   import AeMdw.Ets
   import AeMdw.Util
 
-  ##########
+  @spec name_claim_mutations(Node.tx(), any(), Blocks.block_index(), Txs.txi()) :: [Mutation.t()]
```

Comment:
@sborrazas this looks like a dialyzer limitation. Even if spec returns a `list()` it fails.